modules/UI/util/UIUtil.js

```diff
@@ -2,6 +2,17 @@
 
 import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
 
+const TOOLTIP_POSITIONS = {
```

Comment:
We require jsdoc comments on everything! For example, describe what the keys and the values stand for.